fetch.bs

```diff
@@ -1055,6 +1083,9 @@ must be an algorithm accepting an exception.
    <dd>
     <ol>
      <li><p>Let <var>continueAlgorithm</var> be null.
+     <li><p>If <var>timingInfo</var> is not null, and <var>timingInfo</var>'s
+     <a for="fetch timing info">response start time</a> is not zero, set <var>timingInfo</var>'s
```

Comment:
Should we set the response start time when it *is* zero, rather than when it's not?

Reply:
oops